run_qadi_no_timeout.py

```
@@ -0,0 +1,29 @@
+
+import subprocess
+import sys
+import os
+
+# Set environment variable to disable any potential timeouts
+os.environ['PYTHONUNBUFFERED'] = '1'
+
+# Run the QADI demo without any timeout
+try:
+    # Use sys.executable to ensure we use the same Python interpreter
+    result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)